livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamChannels.kt

```diff
@@ -2,75 +2,60 @@ package io.getstream.livestream.compose.ui
-import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.wrapContentHeight
 import androidx.compose.foundation.lazy.GridCells
+import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.LazyVerticalGrid
-import androidx.compose.foundation.shape.RoundedCornerShape
-import androidx.compose.material.Card
+import androidx.compose.foundation.lazy.items
 import androidx.compose.material.ExperimentalMaterialApi
-import androidx.compose.material.Icon
-import androidx.compose.material.Text
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.rounded.ViewStream
-import androidx.compose.material.ripple.rememberRipple
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.LaunchedEffect
-import androidx.compose.runtime.remember
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
-import androidx.compose.ui.res.colorResource
-import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.api.models.QuerySort
+import io.getstream.chat.android.client.models.Filters
 import io.getstream.chat.android.client.models.name
-import io.getstream.chat.android.compose.ui.theme.ChatTheme
 import io.getstream.chat.android.compose.viewmodel.channel.ChannelListViewModel
-import io.getstream.livestream.compose.R
+import io.getstream.chat.android.compose.viewmodel.channel.ChannelViewModelFactory
+import io.getstream.chat.android.offline.ChatDomain
 import io.getstream.livestream.compose.models.LiveStreamChannelItem
 import io.getstream.livestream.compose.models.LiveStreamType
-import io.getstream.livestream.compose.models.LiveStreamType.*
 import io.getstream.livestream.compose.randomArtWork
 import io.getstream.livestream.compose.randomDescription
 
 /**
- * View component to add custom live stream channels screen
+ * View component to add custom live stream channels screen.
  *
-// * @param context - Local context for fetching resources to draw Icons or vectors
  * @param channelListViewModel - Stream channel list ViewModel to bind all channel data to custom UI provided here.
  * @param modifier - Modifier for styling.
  * @param isDarkTheme - a Boolean state to get Theme is dark or light , used for deciding card background
- * @param columnCount - a toggle State to switch between 2 column grid to single column list
+ * @param isGrid - a toggle State to switch between 2 column grid to single column list
```

Comment:
Why don't we have the `columnCount` anymore? Why is it hardcoded 2 columns or 1?

Reply:
Had removed it to keep `isGrid` or `isList` as priority 

unless stated in design im defaulting to 2 - added it as a function parameter now (updated)